src/components/stops/Search/Search.js

```diff
@@ -0,0 +1,169 @@
+
+  /**
+   * A function which will receive the searchService instance and needs to return a render function for the section title, see [react-autosuggest documentation](https://github.com/moroshko/react-autosuggest#render-section-title-prop) for details.
+   */
+  getRenderSectionTitle: PropTypes.func,
+
+  /**
+   * Props for the search input field, see [react-autosuggest documentation](https://github.com/moroshko/react-autosuggest#input-props-prop) for details.
+   */
+  inputProps: PropTypes.object,
+
+  /**
+   * Callback function which will be called with the hovered suggestion.
+   */
+  onHighlight: PropTypes.func,
+
+  /**
+   * Function to define whether the suggestions are displayed or not.
+   * See 'shouldRenderSuggestions' in [react-autosuggest documentation](https://github.com/moroshko/react-autosuggest#input-props-prop) for details.
+   */
+  shouldRenderSuggestions: PropTypes.func,
+
+  /**
+   * Callback function which will be called with the selected suggestion.
+   */
+  onSelect: PropTypes.func,
+
+  /**
+   * CSS class of the component.
+   */
+  className: PropTypes.string,
+
+  /**
+   * Key to access the engine api.
+   */
+  apiKey: PropTypes.string,
+};
+
+const defaultProps = {
+  apiKey: null,
+  engines: { stops: new StopFinder() },
+  getRenderSectionTitle: () => () => null,
+  onHighlight: () => null,
+  shouldRenderSuggestions: (newValue) => newValue.trim().length > 2,
+  onSelect: () => null,
+  className: 'rt-search',
+  inputProps: {},
+};
+
+function Search({
+  apiKey,
+  engines,
+  getRenderSectionTitle,
+  inputProps,
+  onHighlight,
+  shouldRenderSuggestions,
+  onSelect,
+  className,
+}) {
+  const [suggestions, setSuggestions] = useState([]);
+  const [value, setValue] = useState('');
+
+  const searchService = useMemo(
+    () => new SearchService({ apiKey, engines, setSuggestions }),
+    [apiKey, engines, setSuggestions],
+  );
+
+  const theme = useMemo(() => {
+    return {
+      container: `${className}__container`,
+      containerOpen: `${className}__container--open`,
+      input: `${className}__input`,
+      inputOpen: `${className}__input--open`,
+      inputFocused: `${className}__input--focused`,
+      suggestionsContainer: `${className}__suggestions-container`,
+      suggestionsContainerOpen: `${className}__suggestions-container--open`,
+      suggestionsList: `${className}__suggestions-list`,
+      suggestion: `${className}__suggestion`,
+      suggestionFirst: `${className}__suggestion--first`,
+      suggestionHighlighted: `${className}__suggestion--highlighted`,
+      sectionContainer: `${className}__section-container`,
+      sectionContainerFirst: `${className}__section-container--first`,
+      sectionTitle: `${className}__section-title`,
+    };
+  }, [className]);
+
+  return (
+    Object.keys(engines).length > 0 && (
+      <div className="rt-search">
+        <Autosuggest
+          theme={theme}
+          inputProps={{
+            autoFocus: true,
```

Comment:
> The deploy preview always starts at the Search. @oterral, do you have an idea, why?

> No idea. It's weird.

`autoFocus: false,`

would solve it.

@oterral Should we make it configurable?